addons/web/static/src/js/core/dom.js

```diff
@@ -595,7 +604,8 @@ var dom = {
                     options.progress.apply(this, ...arguments);
                 }
                 const newScrollTop = _computeScrollTop();
-                if (Math.abs(newScrollTop - originalScrollTop) <= 1.0 && !(el.classList.contains('o_transitioning'))) {
+                if (Math.abs(newScrollTop - originalScrollTop) <= 1.0 &&
+                    (isTopOrBottomHidden || !(el.classList.contains('o_transitioning')))) {
```

Comment:
I let developers do as they prefer but FYI I would do it that way (change only if you prefer, I don't mind either way):

```suggestion
                if (Math.abs(newScrollTop - originalScrollTop) <= 1.0
                        && (isTopOrBottomHidden || !(el.classList.contains('o_transitioning')))) {
```
- operators `&&` at the start of the line
- double indent for multi-lined `if`: that way you cannot confuse it with the following instruction